main.go

```diff
@@ -266,19 +266,19 @@ func startEndFromGranularity(t time.Time, granularity string, targetTimezone str
 }
 
 type payload struct {
-	InputSchemaName string `json:"schema"`
-	InputTables     string `json:"tables"`
-	InputBucket     string `json:"bucket"`
-	Truncate        bool   `json:"truncate"`
-	Force           bool   `json:"force"`
-	DataDate        string `json:"date,required"`
-	ConfigFile      string `json:"config"`
-	GZip            bool   `json:"gzip"`
-	Delimiter       string `json:"delimiter"`
-	TimeGranularity string `json:"granularity,required"`
-	StreamStart     string `json:"streamStart"`
-	StreamEnd       string `json:"streamEnd"`
-	TargetTimezone  string `json:"timezone"`
+	InputSchemaName string `config:"schema"`
```

Comment:
This json->config rename is an artifact of the Clever/configure package? I'm not too familiar with it.